sbatches/farmshare/singularity-exec.sbatch

```diff
@@ -17,4 +17,6 @@ fi
 cd $NOTEBOOK_DIR
 
 export SINGULARITY_CACHEDIR=/farmshare/user_data/${USERNAME}/.singularity
+. /etc/profile
+module load singularity/3.4.0
```

Comment:
Do you want to load a specific version here or just singularity to get the latest?

Reply:
l would keep it and load the specific version of 3.4.0, seems to work best for stability wise.